File: src/de/sciencecomputing/venuside/psi/ContextElementType.java
package de.sciencecomputing.venuside.psi;

import com.intellij.psi.tree.IElementType;
import de.sciencecomputing.venuside.ContextLanguage;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.NonNls;

public class ContextElementType extends IElementType {
    public ContextElementType(@NotNull @NonNls String debugName) {
        super(debugName, ContextLanguage.INSTANCE);
    }
}

File: src/de/sciencecomputing/venuside/psi/ContextElementType.kt
package de.sciencecomputing.venuside.psi

import com.intellij.psi.tree.IElementType
import de.sciencecomputing.venuside.ContextLanguage
import org.jetbrains.annotations.NonNls

class ContextElementType(@NonNls debugName: String) : IElementType(debugName, ContextLanguage)

